src/sentry/static/sentry/app/views/organizationStats/index.jsx

```diff
@@ -92,7 +92,7 @@ const OrganizationStats = React.createClass({
       });
     });
 
-    _.each(this.state.rawProjectData, statName => {
+    $.each(this.state.rawProjectData, statName => {
```

Comment:
> couldn't se just use .forEach at this point?

If it's an array yes. If you're only working with arrays then you also don't need the `"collections"` feature set.